### DISCUSSION TITLE
XYZ Plot for results

### DISCUSSION THREAD

**Discussion by simartem** [Jonseed/ComfyUI-Detail-Daemon#5] [Q&A]
There are a lot of input parameters, it is hard to understand the connection in background which these numbers affect.
Can somebody please provide an xyz plot for the most significant input parameters of the node, to plot the results to see what it affects in reality.

Thank you for the great project!

**Comment by Jonseed** [comment]
Thanks. Yes, there are a lot of input parameters in Detail Daemon, but most of them just shift where in the generation process the sigmas are adjusted. A few things might help:

- there is a description of the parameters in the repo README
- the most important parameter is the first one, the `detail_amount`.
- The `start` and `end` values are next most important, as they specify when to start the adjustment and when to end it, during generation.
- the **Graph Sigmas** node can help you see how each of the parameters adjust the sigmas throughout the generation process
- the **Lying Sigmas** node is a simplified version of Detail Daemon, with just an amount (`dishonesty_factor`) and `start` and `end` values. Much easier to get started with.
- **Multiply Sigmas** is even easier with just one parameter. It will multiply all sigmas by that factor, so it works a bit different from Detail Daemon and Lying Sigma, but has a similar effect.

With so many parameters, it's a bit difficult to do an XYZ plot, and it really changes depending on your prompt, seed, and model. The examples given in the README show the effect of one setup of each node. The example JSON workflow also lets you test out all the nodes yourself to compare them, and see what effect adjusting the values has.

I hope that helps.

**Comment by Kaleidia** [comment]
For the detail amount, I uploaded grids or x plots on the question about HiDream, there you can at least see that values influence (at least on HiDream)

**Reply by Kaleidia** [reply]
Made some test on Flux and and the detail amount value, between 0.25 and 0.35 it just adds detail, afterwards it adds more and over 0.55 is goes a bit too crazy for my taste. Beyond 0.75 it starts to distort the image too much.

**Reply by Jonseed** [reply]
Thanks for the tests, but please no nudity here.

**Reply by Kaleidia** [reply]
> Thanks for the tests, but please no nudity here.

if I censor it with black bars, would it be ok?

**Reply by Jonseed** [reply]
yes

**Reply by Kaleidia** [reply]
Sorry for the uncensored version ealier
![Detail_amount_grid_flux_censored](https://github.com/user-attachments/assets/106224ab-9b76-4d3e-8287-8462a052c282)
